frontend/ui/widget/screenshoter.lua

```diff
@@ -54,6 +56,41 @@ function Screenshoter:onScreenshot(filename)
     return true
 end
 
+function Screenshoter:chooseFolder()
+    local buttons = {}
+    table.insert(buttons, {
+        {
+            text = _("Choose screenshot directory"),
+            callback = function()
+                UIManager:close(self.choose_dialog)
+                require("ui/downloadmgr"):new{
+                    onConfirm = function(path)
+                        G_reader_settings:saveSetting("screenshot_dir", path .. "/")
+                        UIManager:show(InfoMessage:new{
+                            text = T(_("Screenshot directory set to:\n%1"), path),
+                            timeout = 3,
+                        })
+                    end,
+                }:chooseDir()
+            end,
+        }
+    })
+    table.insert(buttons, {
+        {
+            text = _("Close"),
+            callback = function()
+                UIManager:close(self.choose_dialog)
+            end,
+        }
+    })
+    local screenshot_dir = G_reader_settings:readSetting("screenshot_dir") or DataStorage:getDataDir() .. "/screenshots/"
+    self.choose_dialog = ButtonDialogTitle:new{
+        title = T(_("Current screenshot directory:\n %1"), screenshot_dir),
```

Comment:
Minor final nitpick, but is that space intentional?

Reply:
This is parametr of `ButtonDialogTitle` so `title=` should be indent.

Reply:
I think he means the one between `\n %1` (which adds some small indent on the 2nd line with the directory path) - but we have it in quite a few places, so either intentional, or it replicated thru cut & paste :)

Reply:
:)